src/__test__/components/data-exploration/generic-gene-table/LaunchPathwayAnalysisModal.test.jsx

```diff
@@ -60,15 +59,13 @@ const pantherDBResponse = {
 const renderPathwayAnalysisModal = async (filtersApplied = false) => {
   await act(async () => {
     render(
-      <SWRConfig value={{ provider: () => new Map() }}>
```

Comment:
What was this `SWRConfig` for?

Reply:
This was a left-over from the previous implementation to get PantherDB species. I used `useSWR` previously for fetching pantherDB species from their API (http://pantherdb.org/services/openAPISpec.jsp). It worked in local, but failed in staging (HTTP vs HTTPS). So we do not `useSWR` any more, and just store the species list in a JSON.

`useSWR` caches results, which is a problem in the tests. This `SWRConfig` is used to clear out cache in tests (https://swr.vercel.app/docs/advanced/cache#reset-cache-between-test-cases). However, because we are not using `useSWR` anymore, I removed the `useSWR` implementation in .